Database/DatabaseSender.py

```diff
@@ -39,7 +40,7 @@ def create_courses(self):
         Creates courses table.
         """
         # TODO - remove antireq field
-        command = "CREATE TABLE IF NOT EXISTS " + self.course_table + """ (
+        command = "CREATE TABLE " + self.course_table + """ (
```

Comment:
similar to my concerns before - what if we had a table already

Reply:
See above